src/init.jsx

```diff
@@ -10,19 +10,23 @@ import socketInit, { socketMessageHook } from './store/socket_init.js';
 // Reducers
 import chat from './store/Chat';
 import ui from './store/UI';
+import config from './store/Config';
 import socket from './store/Socket';
 
 // Create redux store
 const store = createStore(
   combineReducers({
     chat,
     ui,
+    config,
     socket,
   }),
 
   applyMiddleware(socketMessageHook),
 );
 
+// TODO: use a env var to disable this on build
+window.gimmeStore = store.getState;
```

Comment:
Should we be using global scope for this? This doesn't feel like a permanent solution. But I'm not 100% sure.

Reply:
This is just for debugging - it will eventually be disabled via an env variable for development mode.